packages/ui-billboard/src/Billboard/index.tsx

```diff
@@ -23,7 +23,7 @@
  */
 
 /** @jsx jsx */
-import { Component } from 'react'
+import { Component, MouseEvent } from 'react'
```

Comment:
On line 49, add `@tsProps` to the annotation. We have to add it to each component we fully typed so the doc generation knows that it can render the props from the TS types safely instead of the old PropTypes :)

```js
/**
---
category: components
---
@tsProps
**/
```